src/families/bitcoin/wallet-btc/crypto/secp256k1.ts

```diff
@@ -0,0 +1,420 @@
+/* eslint-disable */
+const BigInt = require('big-integer');
```

Comment:
`big-integer` is currently available only through other dependencies, maybe we should add it to package.json?

Also I think we often had issues with bigint polyfills in the past... can you please make sure to test on both Android and iOS?